### ISSUE TITLE
about google cloud storage

### ISSUE THREAD

**Issue by liusy58** [apache/openwhisk#5161]
hi, sir, sorry to disturb you again, I wonder is there any way for me to use google cloud storage or any other service like this on openwhisk?

**Comment by rabbah** [comment]
Yes of course (we provide Google Cloud Storage integration with OpenWhisk at Nimbella for example). What are you trying to do? You might be able to reuse our open sources libraries for this.
